projects/ion/src/lib/chip/chip.component.html

```diff
@@ -3,7 +3,8 @@
   [disabled]="disabled"
   [ngClass]="{
     chip: !selected,
-    'chip-selected': selected
+    'chip-selected': selected,
+    positionIcon: iconPosition == 'right' ? '' : 'positionIcon'
```

Comment:
Try to use triple equals instead of double when you can.
Since you want that 'positionIcon' is setted only when iconPositition isn't right, you can do without the ternary, like this:

```suggestion
    positionIcon: iconPosition !== 'right'
```

or

```suggestion
    positionIcon: iconPosition === 'left'
```

The second way is the best because it makes easier to understand.